submissions/models.py

```diff
@@ -33,6 +33,11 @@
     providing_args=['anonymous_user_id', 'course_id', 'item_id']
 )
 
+# In order to allow for effective soft-deletes, we want to override the default Manager for several object types
+class SoftDeletedManager(models.Manager):
+    def get_queryset(self):
+        #By default, filter out any soft-deleted records for classes using this manager
+        return super(SoftDeletedManager, self).get_queryset().filter(deleted=False)
```

Comment:
When we export to researchers we just dump the data directly out of the database using a MySQL CLI so it won't use the model.

I'm not sure about the downloadable reports. @ormsbee might know?